src/components/WeatherWidget/WeatherWidget.stories.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import { Meta } from '@storybook/react'
+import { ObjectId } from 'bson'
+import WeatherWidget from './WeatherWidget'
+import { WeatherWidget as WeatherWidgetType } from 'types'
+
+const mockWeatherWidget: WeatherWidgetType = {
+  _id: new ObjectId(),
+  type: 'Weather',
+  title: 'Weather',
+  coords: {
+    lat: 34.0901,
+    long: -118.4065,
+    forecastUrl: 'https://api.weather.gov/gridpoints/LOX/149,48/forecast',
+    hourlyForecastUrl:
+      'https://api.weather.gov/gridpoints/LOX/149,48/forecast/hourly',
+    city: 'Beverly Hills',
+    state: 'CA',
+    zipcode: '90210',
+  },
+}
+
+const mockWeatherWidgetNoForecast: WeatherWidgetType = {
+  _id: new ObjectId(),
+  type: 'Weather',
+  title: 'Weather',
+  coords: {
+    lat: 34.0901,
+    long: -118.4065,
+    forecastUrl: '',
+    hourlyForecastUrl: '',
+    city: 'Beverly Hills',
+    state: 'CA',
+    zipcode: '90210',
+  },
+}
+
+export default {
+  title: 'Components/WeatherWidget',
+  component: WeatherWidget,
+} as Meta
+
+export const AddWeatherWidget = () => <WeatherWidget />
+
+export const WeatherWidgetWithData = () => (
+  <WeatherWidget widget={mockWeatherWidget} />
+)
```

Comment:
Suggestion: We probably should mock out the api call to weather.gov in storybook like we do with the RSS widget. Otherwise we'll have a happo diff every time the weather changes. 

Question: should we add mocks to show some of the other weather possibilities?

Reply:
I added code to do this as discussed.